cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -87,14 +86,20 @@ def _do_cfe_civil_check(self):
 
     @staticmethod
     def _is_data_complete_enough_to_call_cfe(case_data):
-        if not hasattr(case_data, "facts"):
+        """
+        The CFE means calculation doesn't always need all the case_data fields known. Here we check for the presence of certain fields.
+        We can't use hasattr(case_data, "<field_name>") to know if a field has been set, because hasattr() under the hood calls getattr(),
```

Comment:
Sorry to niggle, but the docstring is written for callers, which is what L90 is. However this L91-L92 explains implementation, so should instead be a comment